### DISCUSSION TITLE
Shortcut for Resizing Brush

### DISCUSSION THREAD

**Discussion by steveman1123** [PintaProject/Pinta#794] [Ideas]
I am using v2.1.2

I find the workflow to resize brush very clunky (having to move the cursor off the image, click a bunch of times, then go back to the image to see if the brush is the right size).  
  
I'd like to see a keyboard shortcut like Paint.NET where "[" and "]" decrease/increase respectively.

**Comment by cameronwhite** [comment]
That sounds reasonable to me - I've moved this to issue #796

**Reply by cameronwhite** [reply]
This was implemented by #1155
